docker/junebug.conf

```diff
@@ -0,0 +1,5 @@
+command=/scripts/junebug-entrypoint.sh
```

Comment:
Sorry, I updated my comment a few minutes after posting it... This should start with a `[program:...]` line. See the [docs](http://supervisord.org/running.html#adding-a-program).